docs/scos/user/features/201811.0/gift-cards-feature-overview.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Gift Cards feature overview
+description: This article describes general concepts of gift cards, gift card’s purchase and redeeming process, as well as the various use case scenarios.
+originalLink: https://documentation.spryker.com/v1/docs/gift-cards
+originalArticleId: 4fbb59e9-a575-4fbb-867f-a3379210fc68
+redirect_from:
+  - /v1/docs/gift-cards
+  - /v1/docs/en/gift-cards
+  - /v1/docs/gift-card-purchase-management
+  - /v1/docs/en/gift-card-purchase-management
+  - /v1/docs/gift-cards-purchase-redeeming
+  - /v1/docs/en/gift-cards-purchase-redeeming
+---
+
+The _Gift Card_ feature allows you to create a virtual product with a chosen value amount. The purchase of a gift card generates an individual code that can then be used as a payment method during checkout.
+
+A Gift Card is a prepaid certificate entitling its owner to purchase products for the gift card’s value.
+
+When paying for orders with a gift card, the gift cards are considered as an additional payment method. You can use the full balance  or part of the total gift card's value. In the latter case, the remaining value is stored as leftover balance on the original code. You can also use additional gift cards to pay for products. However, gift cards can only be applied for products in the same currency they have been bought.
```

Comment:
In the first clause, `gift card` is singular while in the second one it's plural. Unify it

Reply:
`a leftover balance`

Reply:
The meaning of the `original code` is not clear. Do you mean the used gift card?

Reply:
```suggestion
You can also use multiple gift cards to pay for a product.
```